### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by kianmeng**
Found via:
- `codespell -L observ,datas,nd,vai`
- `typos --format brief`

### CODE REVIEW

**Comment by szepeviktor** [comment]
There is one more: `test_otho_camera_near_far`

**Comment by szepeviktor** [comment]
"datas" could be "cubemap" or "coords"

File: examples/feature_demo/pbr.py [hunk 1]
```diff
@@ -57,7 +57,7 @@
 scene.add(background)
 
 # Load meshes, and apply env map
-# Note that this lits the helmet already
+# Note that this list the helmet already
```
Comment (Korijn) [inline examples/feature_demo/pbr.py:60]: This correction is wrong, should be:

```suggestion
# Note that this lights the helmet already
```

File: pygfx/cameras/_orthographic.py [hunk 2]
```diff
@@ -13,7 +13,7 @@ class OrthographicCamera(PerspectiveCamera):
         may be wider if the viewport is relatively wide.
     height: float
         The (minimum) height of the view-cube. The actual view
-        may be heigher if the viewport is relatively heigh.
+        may be higher if the viewport is relatively height.
```
Comment (Korijn) [inline pygfx/cameras/_orthographic.py:16]: Wrong again, should be:

```suggestion
        may be taller if the viewport is relatively high.
```

File: pygfx/renderers/wgpu/_renderer.py [hunk 3]
```diff
@@ -357,7 +357,7 @@ def _set_wobject_pipelines(self):
         # Renderers with the same blend mode can safely share these
         # wobject_pipeline dicts. Therefore, we make use of a global
         # collection. Since this global collection is a
-        # WeakValueDictionary, if all renderes stop using a certain
+        # WeakValueDictionary, if all renders stop using a certain
```
Comment (Korijn) [inline pygfx/renderers/wgpu/_renderer.py:360]:
```suggestion
        # WeakValueDictionary, if all renderers stop using a certain
```

**Comment by Korijn** [comment]
Please adjust the PR according to the review comments and we can merge.

Very nice, thank you!

File: pygfx/controllers/_orbit.py [hunk 4]
```diff
@@ -69,8 +69,8 @@ def _update_rotate(self, delta):
         delta_azimuth, delta_elevation = delta
         camera_state = self._get_camera_state()
 
-        # Note: this code does not use la.vec_euclidian_to_spherical and
-        # la.vec_spherical_to_euclidian, because those functions currently
+        # Note: this code does not use la.vec_euclidean_to_spherical and
+        # la.vec_spherical_to_euclidean, because those functions currently
```
Comment (almarklein) [inline pygfx/controllers/_orbit.py:73]: This is actually spelled as the method is currently called (see https://github.com/pygfx/pylinalg/blob/main/pylinalg/vector.py#L386). If anything, we should update in pylinalg.

**Review by almarklein — COMMENTED** [review]
Thanks a lot!

Nice to see how class and methods names are consistently renamed. As far as I can tell this fortunately only affects private API, except for that pylinalg one.

File: pygfx/controllers/_orbit.py (hunk 4 above)
Comment (almarklein) [reply pygfx/controllers/_orbit.py:73]: https://github.com/pygfx/pylinalg/pull/84

**Review by almarklein — DISMISSED** [review]
LGTM.

I'm not entirely sure about adding the `typos.toml` though. I think we should only add if we also add a CI job for it (typos has a GH action: https://github.com/crate-ci/typos).

**Review by almarklein — DISMISSED** [review]

**Review by Korijn — APPROVED** [review]

**Comment by Korijn** [comment]
CI failure:

```
/home/runner/work/pygfx/pygfx/docs/_gallery/other/collection_line.rst:22: WARNING: Title underline too short.

Performance of drawing many Lines
================================
```

**Review by Korijn — APPROVED** [review]

**Comment by kianmeng** [comment]
:partying_face: :partying_face: :partying_face: :partying_face: :partying_face:

**Pull request merged**